main/src/ca/uwaterloo/flix/util/collection/Nel.scala

```diff
@@ -30,6 +30,13 @@ case class Nel[T](x: T, xs: List[T]) extends Iterable[T] {
   /** Builds a new [[Nel]] by applying `f` to all elements of `this`. */
   override def map[S](f: T => S): Nel[S] = Nel(f(x), xs.map(f))
 
+  /** Returns two lists from a list of tuples. */
+  override def unzip[A1, A2](implicit asPair: T => (A1, A2)): (Nel[A1], Nel[A2]) = {
```

Comment:
I guess this is how Scala does it for regular lists?

Reply:
Yeah object oriented world is weird